statsmodels/robust/scale.py

```diff
@@ -39,6 +39,8 @@ def mad(a, c=Gaussian.ppf(3/4.), axis=0, center=np.median):
     mad : float
         `mad` = median(abs(`a` - center))/`c`
     """
+    if a.size == 0:
+        return np.array([np.nan])
```

Comment:
Oops, seems return is wrong.  Should be a scalar to be standard with other reductions.  Needs to be fixed before it can be accepted.